tracer/build/_build/Build.Steps.cs

```diff
@@ -706,11 +706,9 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
                 .ToList();
 
             testProjects.ForEach(EnsureResultsDirectory);
-            var filter = (string.IsNullOrEmpty(Filter), IsArm64, IsAlpine) switch
+            var filter = (string.IsNullOrEmpty(Filter), IsArm64) switch
             {
-                (true, true, false) => "(Category!=ArmUnsupported)",
-                (true, false, true) => "(Category!=AlpineUnsupported)",
-                (true, true, true) => "(Category!=AlpineUnsupported)&(Category!=ArmUnsupported)",
+                (true, true) => "(Category!=ArmUnsupported)",
                 _ => Filter
             };
```

Comment:
Not a blocker, but this check is simple enough now that we don't even need a `switch` anymore :) 
```csharp
var filter = string.IsNullOrEmpty(Filter) && IsArm64 ? "(Category!=ArmUnsupported)" : Filter;
```

Reply:
Fixed in 8a119b80f405e64e7f5083658d9d5866e691aa58